### ISSUE TITLE
Enable selecting and copying specific part of the response

### ISSUE THREAD

**Issue by radimjanda754** [ChuckerTeam/chucker#1115]
Currently in response, Im able to select and copy only one line of the response, the cursor wont let me select more. Especially for large responses it would be great to be able to select and copy out multiple lines.
Right now the only option to do this is to save the response as file and then find and copy it from file, however its not so much user friendly, also Android emulators defaultly doesnt have any text editor that could open file exported from chucker.

**Comment by cortinico** [comment]
This is because every line is rendered separately inside a RecyclerView, for performance reasons. I'm unsure if we can allow selections to span multiple TextViews

As you said, the alternative would be to save the response to file and consume it from there.

**Comment by radimjanda754** [comment]
Would it at least by possible to copy whole response to clipboard? Similary like saving whole response to file. I just want to see response in some text editor on computer and work with text from there then, copy paste all text seems as most convinient way to do it. Saving response brings several unnecessary extra steps as you need to move file from emulator, etc

**Comment by cortinico** [comment]
yup copying to clipboard is an option that we could add to the sharing menu. Are you up for sending a PR to do so?
